pkg/builder/local_build_executor_test.go

```diff
@@ -584,7 +585,11 @@ func TestLocalBuildExecutorSuccess(t *testing.T) {
 	inputRootCharacterDevices := map[path.Component]int{
 		path.MustNewComponent("null"): 259,
 	}
-	localBuildExecutor := builder.NewLocalBuildExecutor(contentAddressableStorage, buildDirectoryCreator, runner, clock, time.Hour, time.Hour, inputRootCharacterDevices, 10000)
+	environmentVars := map[string]string{
```

Comment:
Great! Thanks for testing both the behaviour of plain assignment and overwriting an existing one.